operations/helm/charts/mimir-distributed/values.yaml

```
@@ -2976,7 +2978,9 @@ gateway:
           proxy_read_timeout 300;
           server {
             listen {{ include "mimir.serverHttpListenPort" . }};
+            {{- if .Values.gateway.nginx.config.listenIPv6 }}
```

```suggestion
            {{- if .Values.gateway.nginx.config.enableIPv6 }}
```

leaving it here so it's easier to accept if you agree